Cargo.toml

```diff
@@ -2479,6 +2479,17 @@ description = "Systems run in parallel, but their order isn't always determinist
 category = "ECS (Entity Component System)"
 wasm = false
 
+[[example]]
+name = "observers"
+path = "examples/ecs/observers.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.observers]
+name = "Observers"
+description = "Define observers to react to ECS events"
```

Comment:
```suggestion
description = "Observers listen for and react to ECS life-cycle events, such as component addition and removal"
```

Reply:
"life-cycle" is half-truth. The default set of triggers is life-cycle related, but users can define their own, totally unrelated triggers

Reply:
Oh, that's a useful feature. Definitely needs to be more clearly documented but this isn't the right suggestion then.